e2e/app/component/modal.ts

```diff
@@ -8,6 +8,7 @@ import {savePageToFile, takeScreenshot} from 'utils/save-file-utils';
 import {LinkText} from 'app/text-labels';
 import {getPropValue} from 'utils/element-utils';
 import * as fp from 'lodash/fp';
+import {waitWhileLoading} from 'utils/test-utils';
```

Comment:
This appears to ahve created a circular dependency, resulting in an error importing `Modal` from `ConceptSetSaveModal`. moving waitWhileLoading into waits-utils may be the easiest path forward.

Longer term, I suggest removing any dependency from utils -> Page, or components -> Page. This seems like it will just cause more of these issues going forwards.

Reply:
Fixed in https://github.com/all-of-us/workbench/pull/4151

Reply:
Sorry. I should have seen it. Thank you @jmthibault79